test/passing/tests/js_to_do.ml.ref

```diff
@@ -14,7 +14,8 @@ let _ =
 (* js-type *)
 
 (* The following tests incorporate several subtle and different indentation
-   ideas. Please consider this only a proposal for discussion, for now.
+   ideas. Please consider this only a proposal for discussion, for
+   now.
```

Comment:
This is a regression.

Reply:
I'm stuck on this one. The same thing happen in ocamlformat's source: https://github.com/ocaml-ppx/ocamlformat/pull/2371#discussion_r1219329872

The hack required to print an empty line without trailing space is not the same at a point that would break (if the empty line was swapped with a `@ `) and a point that would not break (several small lines).

Reply:
This should be fixed in a400535